src/main/java/io/phdata/streamliner/schemadefiner/util/TemplateFunction.java

```diff
@@ -0,0 +1,150 @@
+package io.phdata.streamliner.schemadefiner.util;
```

Comment:
TemplateFunction class is used in MDT template. Hence added this class and few methods that are used in template.